src/GraphQL/Execution/DefaultExecutionStrategySelector.cs

```diff
@@ -0,0 +1,52 @@
+using GraphQLParser.AST;
+
+namespace GraphQL.Execution
+{
+    /// <inheritdoc cref="IExecutionStrategySelector"/>
+    public class DefaultExecutionStrategySelector : IExecutionStrategySelector
+    {
+        private readonly ExecutionStrategyRegistration[] _registrations;
+
+        /// <summary>
+        /// Initializes an instance that only returns the default registrations;
+        /// <see cref="ParallelExecutionStrategy"/> for <see cref="OperationType.Query"/> and
+        /// <see cref="SerialExecutionStrategy"/> for <see cref="OperationType.Mutation"/>.
+        /// </summary>
+        internal DefaultExecutionStrategySelector()
```

Comment:
60/40 to make it public.

Reply:
Whatever you think is fine with me.  It is not really needed by anything; anyone can always call the other constructor with an empty list.  It's here because it's better documented here than if I was to embed the code directly into `DocumentExecuter`.

Reply:
Let there be public...

Reply:
ok